### DISCUSSION TITLE
是否考虑添加无需登录即可评论的评论系统？

### DISCUSSION THREAD

**Discussion by ycloong** [tangly1024/NotionNext#259] [Ideas]
大佬你好，非常喜欢你的项目。想问一下是否考虑添加新的评论系统呢。现在的评论系统基本上都是需要登录的，而cusdis不需要登录，但是评论的内容是需要管理员审核才能够发布的。那么能不能添加一个评论系统能做到既不需要登录，也能实时展示出来的呢？就像valine一样。

**Comment by tangly1024** [comment]
感谢你的提议。
在后续的版本中，会添加一个valine的评价插件。

**Comment by tangly1024** [comment]
Valine评论插件已添加，更新至3.3.7版本即可体验

体验地址： https://preview.tangly1024.com/article/example-1#comment

关联issues https://github.com/tangly1024/NotionNext/issues/262
关联版本 https://github.com/tangly1024/NotionNext/releases/tag/v3.3.7

**Reply by wopanda** [reply]
作者您好，感谢您添加Valine评论插件，我是今天才部署的3.37版，其他功能均能正常使用，可是这个评论功能我目前尚不能正常体验到，请问是需要再做其他设置吗？

**Reply by tangly1024** [reply]
请参考此文，添加Valine配置 https://tangly1024.com/article/notionnext-valine

**Comment by tangly1024** [comment]
另外一个好消息是，除了Valine插件，我同时加入了Waline的评论插件，功能更加丰富、并配置了完善的后台管理；感兴趣的话也可以更新V3.3.8体验。

体验地址： https://preview.tangly1024.com/article/example-1

关联提交：https://github.com/tangly1024/NotionNext/pull/271
关联版本： https://github.com/tangly1024/NotionNext/releases/tag/v3.3.8

**Reply by tangly1024** [reply]
相关配置教程
https://waline.js.org/guide/get-started.html 

https://tangly1024.com/article/notionnext-valine

**Reply by spacding** [reply]
请问Waline国际版和国内版有什么区别吗？

**Reply by tangly1024** [reply]
> 请问Waline国际版和国内版有什么区别吗？

waline本身不区分国内和国际版，但配置的leancloud数据存储有区分。

leancloud[国内版](https://leancloud.cn/)和[国际版](https://leancloud.app/) 功能上基本没有区别：但国内版在大陆的访问速度可能会更快
